stdlib/list.mli

```diff
@@ -230,6 +230,14 @@ val find_opt: ('a -> bool) -> 'a list -> 'a option
     satisfies [p] in the list [l].
     @since 4.05 *)
 
+val find_map: ('a -> 'b option) -> 'a list -> 'b option
+(** [find_map f l] applies [f] to the elements of [l] in order,
+    and returns the first result of the form [Some v], or [None]
+    if none exist.
+    @since 4.10.0
+*)
```

Comment:
In general I prefer when doc strings describe the result ("`[find_map f l] is Some _` ...) than the operational behaviour ("`returns`" etc.). In that case though my trials were too convoluted and I'm personally fine with this docstring.

Reply:
I agree, and I agree.